lib/clearance/password_strategies/sha1.rb

```diff
@@ -3,13 +3,21 @@ module PasswordStrategies
     module SHA1
       require 'digest/sha1'
 
+      DEPRECATION_MESSAGE = "[DEPRECATION] The SHA1 password strategy " \
+        "has been deprecated and will be removed from Clearance 2.0. BCrypt " \
+        "is the only officially supported strategy, though you are free to " \
+        "provide your own. To continue using this strategy add " \
+        "clearance-deprecated_password_strategies to your Gemfile."
```

Comment:
Common convention is that `-` denotes an extension to whatever precedes it, while `_` is the usual word separator. So since this is a clearance extension, it should be `clearance-` and then follow our usual rules. `factory_girl_rails` should actually be `factory_girl-rails`. I agree that it's ugly, but that's the guide. See: http://guides.rubygems.org/name-your-gem/